candi/bashible/common-steps/all/080_setup_timer_to_run_bashible.sh.tpl

```diff
@@ -40,4 +40,5 @@ Description=Bashible service
 [Service]
 EnvironmentFile=/etc/environment
 ExecStart=/var/lib/bashible/bashible.sh --max-retries 10
+RuntimeMaxSec=3h
```

Comment:
I think it's a very big timeout, why we set it to 3 hours ?

Reply:
because every 4 hours the check amount changes. I decided that this is a reasonable interval. in this format you can catch script hangs and it will work if they are caused by one-time race events.